pkg/daemon/discover/discover.go

```diff
@@ -348,6 +367,18 @@ func probeDevices(context *clusterd.Context) ([]sys.LocalDisk, error) {
 	if err != nil {
 		return devices, fmt.Errorf("failed initial hardware discovery. %+v", err)
 	}
+
+	// ceph-volume inventory command takes a little time to complete.
+	// Get this data only if it is needed and once by function execution
+	var cvInventory *map[string]string = nil
+	if useCVInventory {
+		logger.Infof("Getting ceph-volume inventory information")
+		cvInventory, err = GetCephVolumeInventory(context)
+		if err != nil {
+			logger.Errorf("error getting ceph-volume inventory: %v", err)
```

Comment:
`logger.Errorf("error getting ceph-volume inventory. %v", err)`

btw, shouldn't we return error instead of logging error?

Reply:
In the "probeDevices" function we collect more information than the provided by "ceph-volume". With this implementation, If we have a problem with "ceph-volume" we are going to continue providing all the hardware information. 
Taking into account that other current  "consumers"  do not use the "ceph-volume" information at all, this seems to be the more "healthy" way to deal with a problem in "ceph-volume" and avoid to "break" other "consumers"